istioctl/pkg/multicluster/remote_secret.go

```diff
@@ -527,9 +532,14 @@ func createRemoteSecret(opt RemoteSecretOptions, client kube.ExtendedClient, env
 		return nil, fmt.Errorf("could not get access token to read resources from local kube-apiserver: %v", err)
 	}
 
-	server, err := getServerFromKubeconfig(opt.Context, env.GetConfig())
-	if err != nil {
-		return nil, err
+	var server string
+	if opt.ServerOverride != "" {
+		server = opt.ServerOverride
```

Comment:
It will be good to add validation for the `server` address because the user can pass anything and it can break subsequent usage.

`getServerFromKubeconfig` reads server address directly from `KubeConfig` file so there won't be any validation needed, but if we take input from the user, better to validate it.

Reply:
I'm adding a total config validation, this may catch other things too. Thanks for the suggestion!